modules/nf-core/atlasgeneannotationmanipulation/gtf2featureannotation/main.nf

```diff
@@ -23,6 +23,7 @@ process ATLASGENEANNOTATIONMANIPULATION_GTF2FEATUREANNOTATION {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: meta.id
     def reference_cdna = fasta ? "--parse-cdnas $fasta" : ""
```

Comment:
I think according to the nf-core guidlines only required paramters should get it's own arguments. otherwise, they should be in ext.args.

Reply:
Doesn't apply to file inputs- they all have to come in via channels.

Reply:
true, sorry